American Whitewater/Extensions/Reach.swift

```diff
@@ -51,6 +51,16 @@ extension Reach {
             return ""
         }
     }
+    
+    /// Returns the 'runnability' color for a reaches condition
+    var conditionColor: UIColor {
```

Comment:
Lol, condition is such an ambiguous name. I guess we leave though for consistency.

Reply:
Actually, how hard would it to rename this to `runnabilityColor`?

Reply:
easy

Reply:
Or rather easy, but... I've been trying to refrain from renaming data model stuff, thinking the app is due for an effort to do that to most of the stuff in the models.

Doesn't seem like a problem to rename this as a one-off though, I'll go ahead and do it

Reply:
Yeah I think that's much better. Wasn't sure if it was used more widely.